src/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -127,8 +133,16 @@ public OptionsViewPanel() {
 	private void initialize() {
         this.setLayout(new BorderLayout());
         this.setName(Constant.messages.getString("view.options.title"));
-        this.add(getPanelMisc());
-
+        this.add(getmainScrollPane());
+	}
+	
+	private JScrollPane getmainScrollPane() {

```

Comment:
Sorry if I was misleading before, in the getter the first letter of the name of the variable should be upper case (so `getMainScrollPane`).

Reply:
okay. I will change that.